yarn.lock

```diff
@@ -10751,4 +10751,4 @@ yargs@^16.2.0:
 yocto-queue@^0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/yocto-queue/-/yocto-queue-0.1.0.tgz#0294eb3dee05028d31ee1a5fa2c556a6aaf10a1b"
-  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
+  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
```

Comment:
@Jayakumar03, Here we should have a newline at the end.

Reply:
shall i give two blank line or single blank line is enough ?

Reply:
[This](https://github.com/OpenSource-Journey/Your-Github-Contributions/pull/33/commits/68dc6dc4a7d49f1eb34232411a9dde576aeb1a2c) is good enough.